tests/inference_test.py

```diff
@@ -7,7 +7,7 @@
 
 @pytest.fixture()
 def spacer():
-    return ChatSpace()
+    return ChatSpace(encoding="utf-8")
```

Comment:
```suggestion
    return ChatSpace()
```
default 값이 utf-8 이니깐 굳이 안붙여도 되지 않을까요?

Reply:
default 값이 utf-8이기는 하지만 많은 분들이 encoding을 넣는 부분에서 혼동을 가지고 계시므로 최대한 드러내고자 하였습니다.